watertap/costing/units/electrodialysis.py

```diff
@@ -56,7 +38,7 @@ def build_electrodialysis_cost_param_block(blk):
     build_rule=build_electrodialysis_cost_param_block,
     parameter_block_name="electrodialysis",
 )
-def cost_electrodialysis(blk, cost_electricity_flow=True):
+def cost_electrodialysis(blk, cost_electricity_flow=True, has_rectifier=False):
```

Comment:
I think has_rectifier should default to true, as in general, ED requires an electrifier to operate.  @adam-a-a  @lbibl Thoughts?

Reply:
I don't  really think this matters, as long as the user can read the word... :)